static/Dockerfile

```diff
@@ -1,29 +1,37 @@
-FROM ubuntu:21.04
+# syntax = docker/dockerfile:1.2
 
-ARG DEBIAN_FRONTEND=noninteractive
-ARG TZ=Etc/UTC
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
Same here, remove comment and use `ubi-minimal` ?

Reply:
Probably we should use an UBI as well for consistency :-)

Reply:
Because `ubi-minimal` does not carry `yum` ;-)

```
StanNetwork:instana-agent-docker michele$ (cd dynamic/ && DOCKER_BUILDKIT=1 docker build --secret id=download_key,src=download_key --no-cache . -t instana-agent)
[+] Building 3.2s (10/16)                                                                                                                                                                                       
 => [internal] load build definition from Dockerfile                                                                                                                                                       0.0s
 => => transferring dockerfile: 3.54kB                                                                                                                                                                     0.0s
 => [internal] load .dockerignore                                                                                                                                                                          0.0s
 => => transferring context: 2B                                                                                                                                                                            0.0s
 => resolve image config for docker.io/docker/dockerfile:1.2                                                                                                                                               1.5s
 => [auth] docker/dockerfile:pull token for registry-1.docker.io                                                                                                                                           0.0s
 => CACHED docker-image://docker.io/docker/dockerfile:1.2@sha256:e2a8561e419ab1ba6b2fe6cbdf49fd92b95912df1cf7d313c3e2230a333fdbcc                                                                          0.0s
 => [internal] load metadata for registry.access.redhat.com/ubi8/ubi-minimal:latest                                                                                                                        0.0s
 => [internal] load build context                                                                                                                                                                          0.0s
 => => transferring context: 32.96kB                                                                                                                                                                       0.0s
 => [1/9] FROM registry.access.redhat.com/ubi8/ubi-minimal:latest                                                                                                                                          0.0s
 => => resolve registry.access.redhat.com/ubi8/ubi-minimal:latest                                                                                                                                          0.0s
 => [2/9] ADD docker.repo /etc/yum.repos.d/docker.repo                                                                                                                                                     0.1s
 => ERROR [3/9] RUN --mount=type=secret,id=download_key,dst=/run/secrets/download_key,required=true     curl -sSL https://packages.instana.io/Instana.gpg -o /tmp/Instana.gpg &&     rpm --import /tmp/In  1.1s
------
 > [3/9] RUN --mount=type=secret,id=download_key,dst=/run/secrets/download_key,required=true     curl -sSL https://packages.instana.io/Instana.gpg -o /tmp/Instana.gpg &&     rpm --import /tmp/Instana.gpg &&     echo -e "[instana-agent]\nname=Instana\nbaseurl=https://_:$(cat /run/secrets/download_key)@packages.instana.io/agent/generic/x86_64\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\ngpgkey=https://packages.instana.io/Instana.gpg\nsslverify=1" > /etc/yum.repos.d/Instana-Agent.repo &&     yum install -y --disableplugin=subscription-manager docker-ce-cli iptables iproute util-linux python3 jq instana-agent-dynamic &&     ln -s /opt/instana/agent/licenses /licenses &&     rm -rf /tmp/* /etc/yum.repos.d/Instana-Agent.repo /etc/yum.repos.d/docker.repo &&     yum clean all -y --disableplugin=subscription-manager:
#10 1.014 /bin/sh: yum: command not found
------
executor failed running [/bin/sh -c curl -sSL https://packages.instana.io/Instana.gpg -o /tmp/Instana.gpg &&     rpm --import /tmp/Instana.gpg &&     echo -e "[instana-agent]\nname=Instana\nbaseurl=https://_:$(cat /run/secrets/download_key)@packages.instana.io/agent/generic/x86_64\nenabled=1\ngpgcheck=1\nrepo_gpgcheck=1\ngpgkey=https://packages.instana.io/Instana.gpg\nsslverify=1" > /etc/yum.repos.d/Instana-Agent.repo &&     yum install -y --disableplugin=subscription-manager docker-ce-cli iptables iproute util-linux python3 jq instana-agent-dynamic &&     ln -s /opt/instana/agent/licenses /licenses &&     rm -rf /tmp/* /etc/yum.repos.d/Instana-Agent.repo /etc/yum.repos.d/docker.repo &&     yum clean all -y --disableplugin=subscription-manager]: exit code: 127
```

Reply:
Yeah, but why do we need `yum` ??

Reply:
Basically we go from `141593049` in size to `200832896`. Which is a 50% increase...

Reply:
Pushed a new `microdnf` commit